content/english/phaser-space-invaders-game/activity-10.md

```diff
@@ -9,23 +9,26 @@ Now, we will put our separation of scene 1 and scene 2 to good use by adding a t
 
 This is very simple! we will first comment out `startGame() line` from the bottum of `create()` in `Scene 1`. We are going to use `startGame() fuction` to start game instead of start automatically as before:
```

Comment:
```suggestion
This is very simple! we will first comment out `startGame()` line from the bottom of `create()` in `Scene 1`. We are going to use `startGame()` function to start game instead of start automatically as before:
```